docs/book/src/clusterctl/commands/help.md

```diff
@@ -0,0 +1,4 @@
+# clusterctl help
+
+Help provides help for any command in the application.
+Simply type clusterctl help [path to command] for full details.
```

Comment:
just [command] would be suffice here, path to command would be confusing?

Reply:
Fixed! Thank you for the suggestion